feathr_project/feathr/_databricks_submission.py

```diff
@@ -129,6 +129,7 @@ def submit_feathr_job(self, job_name: str, main_jar_path: str,  main_class_name:
         submission_params['run_name'] = job_name
         submission_params['libraries'][0]['jar'] = self.upload_or_get_cloud_path(main_jar_path)
         submission_params['new_cluster']['spark_conf'] = configuration
+        submission_params['new_cluster']['custom_tags'] = job_tags
         submission_params['spark_jar_task']['parameters'] = arguments
         submission_params['spark_jar_task']['main_class_name'] = main_class_name
```

Comment:
how do we ensure all fields exist?

Reply:
This is like python behavior (if the key doesn't exist then it will be created)